docarray/typing/tensor/audio/abstract_audio_tensor.py

```diff
@@ -20,28 +19,35 @@ def to_bytes(self):
         tensor = (tensor * MAX_INT_16).astype('<h')
         return tensor.tobytes()
 
-    def save_to_wav_file(
+    def save(
         self: 'T',
         file_path: Union[str, BinaryIO],
-        sample_rate: int = 44100,
+        format: str = 'wav',
+        frame_rate: int = 44100,
         sample_width: int = 2,
+        pydub_args: Dict[str, Any] = {},
```

Comment:
can u add the corresponding docstring as well ? Just say that this arg are forwarded to the export function